cmd/nodecmd/create.go

```diff
@@ -170,9 +174,41 @@ func preCreateChecks() error {
 	if customGrafanaDashboardPath != "" && !utils.FileExists(utils.ExpandHome(customGrafanaDashboardPath)) {
 		return fmt.Errorf("custom grafana dashboard file does not exist")
 	}
+
+	if useAWS {
+		if stringToAWSVolumeType(volumeType) == "" {
+			return fmt.Errorf("invalid AWS volume type provided")
+		}
+		if volumeType != constants.DefaultAWSVolumeType && throughput != constants.DefaultAWSThroughput {
```

Comment:
I think it would be clearer if you made this `gp3` explicitly instead of using `DefaultAWSVolumeType`